services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(tickets: list,
+                 username: str,
+                 date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user_id=user.id)
+    if date:
+        order.created_at = date
```

Comment:
Date creates automatically, you have auto_now_add in the model

Reply:
It's rule in task, also if  I delete it from here, 1 test is will be failed